.github/workflows/pr.yaml

```diff
@@ -0,0 +1,19 @@
+name: PR
+
+on:
+  pull_request:
+    types: [opened, ready_for_review, synchronize]
+
+jobs:
+  build:
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go 1.21.x
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.20'
```

Comment:
Did you want to have go 1.20 here when the name of the step is `Setup Go 1.21.x`

Reply:
Ahh good point. Copy pasta fail. I will fix it.